flink-clients/src/main/java/org/apache/flink/client/program/rest/RestClusterClient.java

```diff
@@ -355,7 +368,7 @@ public CompletableFuture<JobResult> requestJobResult(@Nonnull JobID jobId) {
     }
 
     @Override
-    public CompletableFuture<JobID> submitJob(@Nonnull JobGraph jobGraph) {
```

Comment:
we should leave an implementation of public CompletableFuture<JobID> submitJob(@Nonnull JobGraph jobGraph)